src/components/redditClone/index.js

```diff
@@ -2,14 +2,14 @@ import React from 'react'
 import _ from 'lodash'
 import Header from 'components/redditClone/header.js'
 import Posts from 'components/redditClone/posts.js'
-import NewPost from 'components/redditClone/newPost.js'
+import { newPostPage } from 'redux/actions/posts'
 
 export default props => {
   return props.loading === true
     ? <p>Loading posts...</p>
     : <div>
         <Header />
         {_.map(props.posts, post => <Posts post={post} />)}
-        <NewPost onAddPost={props.onAddPost} onUpdatePostLink={props.onUpdatePostLink} onUpdatePostTitle={props.onUpdatePostTitle} />
+        <button onClick={newPostPage}>Add a New Post</button>
```

Comment:
save yourself a bunch of hassle and just make this a link rather than a button.

Delete code appropriately.